www/nodes.js

```diff
@@ -188,16 +188,26 @@ d3.csv(base_url + "cluster.csv", function(error, data) {
           }
       }
     }).then(function() {
-      console.log(data); 
+      console.log(data);
       node = svg.selectAll(".dot")
           .data(data, function(d) { return d.k + '-' + d.topic; })
         .enter().append("circle")
           .attr("class", "dot")
+          .attr("id", function(d) { return d.k + '_' + d.topic; })
           .attr("r", function(d) { return d.radius; })
           .attr("cx", function(d) { return x(d[xVar]); })
           .attr("cy", function(d) { return y(d[yVar]); })
+          .attr("visibility", 'hidden')
           //.attr("style", function(d) { return "fill:" + d.color +"; fill-opacity: "+ opacity(d.opacity) + ";"; })
-          .style("fill", function(d) { return d.color; })
+          .style("fill", function(d) { 
+            var url = base_url + d.k + '/topics.json';
+            d3.json(url, function(topics) {
```

Comment:
recursive call to get all the topics for every single node? I don't know how I missed this during the merge, but this is why there's a delay.